resources/views/auth/passwords/edit.blade.php

```diff
@@ -55,6 +55,7 @@
                                 {{ $errors->first('password') }}
```

Comment:
Can you check without this line? in the layouts/admin, we are checking errors

Reply:
I have made no changes to this file. Just added a blank line accidentally. This line was already present. Should I remove it?

Reply:
ok.. Good. Keep It